src/blosc2/ndarray.py

```diff
@@ -745,9 +772,45 @@ def get_chunk(self, nchunk):
         """Shortcut to :meth:`SChunk.get_chunk <blosc2.schunk.SChunk.get_chunk>`. This can be accessed
         through the :attr:`schunk` attribute as well.
 
+        Parameters
+        ----------
+        nchunk : int
+        The index of the chunk to retrieve.
+
+        Returns
+        -------
+        chunk : array-like
+            The chunk data at the specified index. The type of this data depends on how the chunks are stored and
+            represented in the `SChunk` object.
+
         See Also
         --------
         :attr:`schunk`
+            The attribute that provides access to the underlying `SChunk` object.
+
+
+         Examples
+        --------
+        >>> import blosc2
+        >>> import numpy as np
+        >>> # Create an SChunk with some data
+        >>> data = np.arange(100)
+        >>> schunk = blosc2.SChunk(data=data)
+        >>> # Get a specific chunk from the SChunk object
+        >>> chunk = schunk.get_chunk(0)
+        >>> # Decompress the chunk to convert it into a numpy array
+        >>> decompressed_chunk = blosc2.decompress(chunk)
+        >>> np_array_chunk = np.frombuffer(decompressed_chunk, dtype=np.int64)
+        >>> # Verify the content of the chunk
+        >>> if isinstance(np_array_chunk, np.ndarray):
+        >>>         print(np_array_chunk)
+        >>>         print(np_array_chunk.shape) # Assuming chunk is a list or numpy array
+        [ 0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19 20 21 22 23
+        24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47
+        48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71
+        72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95
+        96 97 98 99]
+        (100,)
```

Comment:
Too many data for an example.  You could explain the same with an array with e.g. 10 values.

Reply:
Done!